src/views/Alteration.vue

```diff
@@ -351,12 +351,23 @@ export default class Alteration extends Mixins(CommonMixin, FeeMixin, FilingTemp
 
     if (items.length !== 7) throw new Error('Failed to fetch entity snapshot')
 
+    // WORK-AROUND WARNING !!!
+    // convert orgPersons from "middleInitial" to "middleName"
+    const orgPersons = items[4].map(orgPerson => {
+      const middleInitial = orgPerson.officer['middleInitial']
+      if (middleInitial !== undefined) {
+        orgPerson.officer.middleName = middleInitial
+        delete orgPerson.officer['middleInitial']
+      }
+      return orgPerson
+    })
```

Comment:
This is the same work-around that was already in place for Change and Conversion filings.

Reply:
Sample:
![image](https://github.com/bcgov/business-edit-ui/assets/10002889/8c1add8b-02a4-4e28-adb8-5405658fed00)